Functions/AddRecord.dart

```diff
@@ -0,0 +1,150 @@
+import 'dart:io';
+import '../Utills/Exception.dart';
+Map<int,Map<String,dynamic>> addingRecording(Map<int,Map<String,dynamic>> data){
```

Comment:
use simple names `addRecord` , should not use present continuous forms.

Reply:
give a block comment over these functions to specify, what params they accept, and what type of data is returned.